tests/licensedcode/data/datadriven/lic1/boto-boto3-cw-example-creating-alarm.txt.yml

```diff
@@ -0,0 +1,5 @@
+license_expressions:
```

Comment:
Should be only `cc-by-nc-sa-4.0`, missing rule.

Reply:
correct. Adding a new rules for cc-by-nc-sa-4.0 such as these would cure this

- `is_license_notice: yes`

```
   This work is licensed under a {{Creative Commons Attribution-NonCommercial-ShareAlike 4.0}}
   International License (the "License"). You may not use this file except in compliance with the
   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.

   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
   either express or implied. See the License for the specific language governing permissions and
   limitations under the License.
```

and `is_license_reference: yes` and `relevance: 100`
```
{{Creative Commons Attribution-NonCommercial-ShareAlike 4.0}} International License
```

Reply:
Nice thanks, fixed by adding these rules. I've added two rules where one is the license notice and the other is the license reference. Adding the license notice fixed the test, is the license reference for completeness?

Reply:
>  is the license reference for completeness?

yes!